opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -269,6 +259,17 @@ message Profile {
   // The field is optional, however if it is present then equivalent converted data should be populated in other fields
   // of this message as far as is practicable.
   bytes original_payload = 21;
+
+  // References to attributes in attribute_table. [optional]
+  // It is a collection of key/value pairs. Note, global attributes
+  // like server name can be set using the resource API. Examples of attributes:
+  //
+  //     "/http/user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36"
+  //     "/http/server_latency": 300
+  //     "abc.com/myattribute": true
+  //     "abc.com/score": 10.239
+  //
+  repeated int32 attribute_indices = 22;
```

Comment:
Do we want to simply reuse the field ID of the removed field maybe? It's not a backward compatible change, but we recently did a field renumbering and I assume before releasing version 1 we'd want to make sure we don't have ID gaps, so we might as well do this now. @jhalliday WDYT

Reply:
For the moment I would vote for just using a new field ID and not reuse the existing one, change its field name and field type. Before declaring this proto as stable, closing these gaps make sense, but I think this will still take a while.

Reply:
SGTM